lib/hydra/remote_identifier/remote_services/doi.rb

```diff
@@ -3,6 +3,7 @@
 require 'hydra/remote_identifier/remote_service'
 require 'hydra/remote_identifier/exceptions'
 require 'active_support/core_ext/hash/indifferent_access'
+require 'byebug'
```

Comment:
Can byebug be removed since we're using the gem?

Reply:
Removed